test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -279,4 +297,44 @@ public void removeLiveCluster(String temp_dir)
         String[] endpoints = result.split(",");
         return endpoints;
     }
+
+    public void ssTableSplit(int node, String options, String keyspace)
+    {
+        String fullCommand;
+        if(options == "")
+        {
+            fullCommand = "ccm node" + node + " sstablesplit" + " --config-dir=" + ccmDir;
+        }
+        else
+        {
+            fullCommand = "ccm node" + node + " sstablesplit " + options + " --config-dir=" + ccmDir;
+        }
+        executeAndPrint(fullCommand);
```

Comment:
You're going to be duplicating --config-dir here, no? Same as in ssTableMetaData() at line 327